test/test_frequencyseries.py

```diff
@@ -540,7 +542,7 @@ def __init__(self, *args):
     for ot in otypes:
         for epoch in epochs:
             na = 'test' + str(i)
-            vars()[na] = test_maker(t, ot, epoch)
```

Comment:
I guess this test suite avoids the `del` by always using `vars()` directly .... Not sure it is neater though.